### ISSUE TITLE
TypeScript interface sorting

### ISSUE THREAD

**Issue by ygrandgirard** [bryanrsmith/eslint-plugin-sort-class-members#95]
Hello,

Is there any reason why this library does not handle TypeScript interfaces?

Simply adding `rules.TSInterfaceDeclaration = rules.ClassDeclaration;` at the end of `sortClassMembersRule.create` in `sort-class-members.js` seems to do the trick. Is there a good reason not to?

**Comment by bryanrsmith** [comment]
I haven't spent enough time in opinionated typescript projects to know. I'm happy to take a PR for it. Is this something that needs to be controlled with a setting? It might be best to consider it a breaking change if that behavior is on by default. Feedback from other TypeScript devs would be great!

**Comment by bryanrsmith** [comment]
Yeah, I agree that it would be a sensible default behavior to have on. If there's a good opportunity for a major version bump in the future I think we can turn it on by default. I'd rather not have a minor release introduce a bunch of rule violations for people.

**Comment by bryanrsmith** [comment]
Resolved by #96, published in v1.19.0. Thanks!
